### PR TITLE
gitignore update

### PR DESCRIPTION
**Pull request by alexproj**
Creating the new branch. gitignore file is updated.

### CODE REVIEW

File: .idea/workspace.xml [hunk 1]
```diff
@@ -270,12 +282,6 @@
     <servers />
```
Comment (byteofmydream) [inline .idea/workspace.xml:282]: Something strange is going on here

File: .gitignore [hunk 2]
```diff
@@ -10,3 +11,4 @@ buildNumber.properties
 # https://github.com/takari/maven-wrapper#usage-without-binary-jar
 .mvn/wrapper/maven-wrapper.jar
 .idea/workspace.xml
+.idea/workspace.xml
```
Comment (byteofmydream) [inline .gitignore:14]: comment1

File: .idea/workspace.xml (hunk 1 above)
Comment (alexproj) [reply .idea/workspace.xml:282]: everything is ok

File: .gitignore (hunk 2 above)
Comment (alexproj) [reply .gitignore:14]: comment2

**Review by byteofmydream — APPROVED** [review]

**Pull request merged**